src/Documentation/sidebar.json

```diff
@@ -115,10 +115,79 @@
       "dvcignore",
       "updating-tracked-files",
       "large-dataset-optimization",
-      "external-dependencies",
       {
-        "label": "Managing External Data",
-        "slug": "managing-external-data"
+        "label": "DVC Remotes",
```

Comment:
let's split remotes into a separate PR. I don't have capacity to review a few PRs of 2000+ lines of code. At this pace of addressing comments they won't even be merged :(

Reply:
Let's keep them in the same PR but focus the review on External Data first, and after we are done with that focus on Remotes.
By splitting them we would lose context and discussions, and it is also a lot of work to undo and redo.